### PR TITLE
Update ZenodoBestPractices.md

### PR DESCRIPTION
**Pull request by ljhwang**
Add information on new versioning feature implemented by zenodo.

### CODE REVIEW

File: ZenodoBestPractices.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ You may use your github account or orcid account as login.
 
 CIG maintains a zenodo community at https://zenodo.org/communities/geodynamics.  Search for: "Computational Infrastructure for Geodynamics" or simply on the word "geodynamics"
 
+Update: zenodo began supporting versioning on May 30, 2078. Please see their blog post and FAQs for more information.
```
Comment (tjesser-ucdavis-edu) [inline ZenodoBestPractices.md]: > May 30, 2078

This probably needs to be fixed.

File: ZenodoBestPractices.md [hunk 2]
```diff
@@ -34,9 +37,15 @@ Assign metadata using the guidelines below.
...
+
 * Click on "Upload" on the browser header (between search bar and *Communities*).
-* Click on "New Upload" - green button on upper right hand side of window.
-Note that your previous data packages are also shown. You can select these and edit their metadata.
+
+
+* **New Package.** Click on "New Upload" - green button on upper right hand side of window.
+* **Versioning a Package.** Select the package from your Uploads directory. Click on "New Version"
+
+At any time, you can select an uploaded package and edit its metadata. However, you CANNOT change the package's files without creating a new version.
```
Comment (tjesser-ucdavis-edu) [inline ZenodoBestPractices.md]: So if I create a new version, I can change existing files?

**Comment by ljhwang** [comment]
sure.

Best,
-Lorraine

*****************************
Lorraine Hwang, Ph.D.
Associate Director, CIG
530.752.3656

File: ZenodoBestPractices.md (hunk 2 above)
Comment (ljhwang) [inline ZenodoBestPractices.md]: yes, this will assign it a new doi and it will only update the files that are changed per the FAQs

**Pull request merged**